database/migrations/2021_05_05_012606_create_area_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateAreaTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('area', function (Blueprint $table) {
+            $table->increments('area_id');
+            $table->string('area_name');
+            $table->timestamps();
+        });
+    }
```

Comment:
Migration must follow laravel convention:
- Table name:  `area` -> `areas`.
- `area_id` -> `id`
- `area_name` -> `name`

Reply:
I change it again.